src/Compiler/Reports/ITaskReportManager.h

```diff
@@ -26,6 +26,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 namespace FOEDAG {
 
+struct TaskMessage {
+  enum class MessageSeverity {
```

Comment:
I think enum class could be moved outside the struct to avoid long access string.

Reply:
agree. fixed